resources/assets/uad_lists.json

```
@@ -15650,7 +15650,7 @@
   },
   "com.mi.health": {
     "list": "Oem",
-    "description": "Heart Rate\nHidden app, not available for users.\nhttps://www.reddit.com/r/Xiaomi/comments/v0uhhx/found_hidden_heart_rate_app_in_miui_1304_i_opened/",
+    "description": "Heart Rate\nHidden app, not available for users.\nhttps://www.reddit.com/r/Xiaomi/comments/v0uhhx/found_hidden_heart_rate_app_in_miui_1304_i_opened/\nIn HyperOS, this app looks like it's done.",
```

```suggestion
    "description": "Heart Rate\nHidden app, not available for users.\nhttps://www.reddit.com/r/Xiaomi/comments/v0uhhx/found_hidden_heart_rate_app_in_miui_1304_i_opened/\nIn HyperOS, this app looks like it's done.",
```
What do you mean with "it's done"? Deprecated? End-of-life perhaps?

---

this app is done and there are no hidden activities like on miui 14: https://www.apkmirror.com/apk/xiaomi-inc/mi-health/mi-health-1-0-6-release/mi-health-1-0-6-android-apk-download/

---

Yeah okay, so you mean it's deprecated. Please mention that in the description. Not many people would understand what you mean with "It's done".